LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -978,8 +978,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/StanfordSpezi/SpeziLLM";
 			requirement = {
-				kind = upToNextMinorVersion;
-				minimumVersion = 0.10.0;
+				branch = "feat/custom-cloud-endpoint";
```

Comment:
TODO: Use tagged SpeziLLM dependency

Reply:
This and #106 are introducing branch based dependencies. I will open an issue so we don't forget to change it later.

Reply:
No need to for this PR, already changed it to use the tagged version. Just wanted to get your reviews first before merging and tagging SpeziLLM v0.11